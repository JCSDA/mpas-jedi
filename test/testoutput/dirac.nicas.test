Test     : Increment norm: 7568.15
Test     : Increment norm: 5291.59
Test     : Adjoint test: 3264.82 / 3264.82

