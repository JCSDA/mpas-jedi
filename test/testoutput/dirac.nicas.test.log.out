Configuration input file is: testinput/dirac.nicas.json
Full configuration is:YAMLConfiguration[path=testinput/dirac.nicas.json, root={Covariance => {covariance => ensemble , date => 2012-05-25T00:00:00Z , localization => {Lbh(1) => 1e+06 , Lbh(2) => 1e+06 , Lbh(3) => 1e+06 , Lbh_file => none , Lbv(1) => 0.4 , Lbv(2) => 0.4 , Lbv(3) => 0.4 , Lbv_file => none , check_adjoints => 1 , check_dirac => 1 , check_mpi => 1 , check_perf => 0 , check_pos_def => 0 , colorlog => 0 , datadir => . , latdir(1) => -31.7 , latdir(2) => 10.7 , latdir(3) => 21.7 , levdir => 1 , levs(1) => 1 , levs(2) => 2 , levs(3) => 3 , localization => NICAS , logpres => 0 , londir(1) => -1.2 , londir(2) => 147 , londir(3) => -79.8 , lsqrt => 0 , mask_check => 0 , model => oops , mpicom => 2 , ndir => 3 , network => 0 , new_mpi => 1 , new_param => 1 , nl => 3 , nproc => 1 , nrep => 2 , ntry => 3 , prefix => mpas , resol => 9 , sam_default_seed => 1} , members => 5 , state => ({date => 2012-05-25T00:00:00Z , filename => Data/x1.40962.init.2012052500.e1.nc},{date => 2012-05-25T00:00:00Z , filename => Data/x1.40962.init.2012052500.e2.nc},{date => 2012-05-25T00:00:00Z , filename => Data/x1.40962.init.2012052500.e3.nc},{date => 2012-05-25T00:00:00Z , filename => Data/x1.40962.init.2012052500.e4.nc},{date => 2012-05-25T00:00:00Z , filename => Data/x1.40962.init.2012052500.e5.nc}) , variables => {variables => onevar}} , dirac => {iCell(1) => 20 , ifdir => 1 , ildir => 1 , ndir => 1} , initial => {date => 2012-05-25T00:00:00Z , filename => Data/x1.40962.init.2012052500.e1.nc , read_from_file => 1} , logging => {categories => Info, Test, Warning, Error} , output_Bens => {datadir => Data , date => 2012-05-25T00:00:00Z , exp => Bens , type => an} , output_nicas => {datadir => Data , date => 2012-05-25T00:00:00Z , exp => dirac , type => an} , resolution => {gridfname => Data/x1.40962.init.2012052500.e1.nc} , variables => {variables => onevar}}]
OOPS_STATS ObjectCountHelper started.
Run: Starting oops::Dirac<MPAS>
Setup resolution OK
SVAR: onevar
Setup variables OK
SVAR: onevar
 mpas_field:read_file: opening Data/x1.40962.init.2012052500.e1.nc
 validity date is: 2012-05-25T00:00:00Z                                            
Setup initial state OK
Setup times OK
Setup increment OK
NICAS setup
Read JSON
Listing setup
Parallel setup:    1 MPI tasks and    4 OpenMP threads
Initialize coordinates
NICAS setup done
Setup localization OK
NICAS multiply
Apply NICAS OK
Test     : Increment norm: 0.0111366
 mpas_field:write_file: writing Data/dirac.an.2012-05-25T00:00:00Z.nc
Write increment OK
